public_html/js/index.js

```diff
@@ -1,35 +1,42 @@
-        buttonClass = 'success';
-        unusedButtonClass = 'danger';
-    } else if ( val == 'false' ) {
-        buttonId = '#danger' + id;
-        unusedButtonId = '#success' + id;
-        unusedButtonClass = 'success';
-        buttonClass = 'danger';
-    }
-    $( buttonId ).removeClass( 'btn-' + buttonClass ).addClass( 'btn-' + buttonClass + '-clicked' ).blur();
-    $( unusedButtonId ).removeClass( 'btn-' + unusedButtonClass ).addClass( 'btn-secondary' ).prop( 'disabled', 'disabled' ).blur();
+	var buttonId, unusedButtonId, buttonClass, unusedButtonClass;
 
-    $.get( 'review/add',
-        { id: id, val: val }
-    ).done( function ( ret ) {
-            console.log( ret );
-            if ( ret == 'false' ) {
-                $( buttonId ).addClass( 'btn-' + buttonClass ).removeClass( 'btn-' + buttonClass + '-clicked' ).blur();
-                $( unusedButtonId ).removeClass( 'btn-secondary' ).prop( 'disabled', false ).addClass( 'btn-' + unusedButtonClass );
-                alert( 'There was an error in connecting to database.' );
-            } else if ( ret == 'Unauthorized' ) {
-                alert( 'You need to be logged in to be able to review.' );
-                $( buttonId ).addClass( 'btn-' + buttonClass ).removeClass( 'btn-' + buttonClass + '-clicked' ).blur();
-                $( unusedButtonId ).removeClass( 'btn-secondary' ).prop( 'disabled', false ).addClass( 'btn-' + unusedButtonClass );
-            }
-        }
-    );
+	if ( val === 'fixed' ) {
+		buttonId = '#success' + id;
+		unusedButtonId = '#danger' + id;
+		buttonClass = 'success';
+		unusedButtonClass = 'danger';
+	} else if ( val === 'false' ) {
+		buttonId = '#danger' + id;
+		unusedButtonId = '#success' + id;
+		unusedButtonClass = 'success';
+		buttonClass = 'danger';
+	}
+	$( buttonId ).removeClass( 'btn-' + buttonClass ).addClass( 'btn-' + buttonClass + '-clicked' ).blur();
+	$( unusedButtonId ).removeClass( 'btn-' + unusedButtonClass ).addClass( 'btn-secondary' ).prop( 'disabled', 'disabled' ).blur();
+
+	$.ajax({
+		url: 'review/add',
+		data: {
+			id: id,
+			val: val
+		},
+		dataType: 'json'
+	}).done( function ( ret ) {
+			if ( ret.user ) {
+				$reviewerNode = $( '.status-div-reviewer-' + id ).show();
+				$reviewerNode.find('.reviewer-link').prop( 'href', ret.userpage ).text( ret.user );
+				$reviewerNode.find('.reviewer-timestamp').text( ret.timestamp );
+				$( unusedButtonId ).removeClass( 'btn-secondary' ).prop( 'disabled', true ).addClass( 'btn-' + unusedButtonClass );
+			} else {
+				if ( ret.error === 'Unauthorized' ) {
+					alert( 'You need to be logged in to be able to review.' );
+				} else {
+					alert( 'There was an error in connecting to database.' );
+				}
+				$( buttonId ).addClass( 'btn-' + buttonClass ).removeClass( 'btn-' + buttonClass + '-clicked' ).blur();
+				$( unusedButtonId ).removeClass( 'btn-secondary' ).prop( 'disabled', false ).addClass( 'btn-' + unusedButtonClass );
+			}
+		}
+	);
```

Comment:
Made spaces into tabs per [coding conventions](https://www.mediawiki.org/wiki/Manual:Coding_conventions/JavaScript#Whitespace) (unclear if they apply here, but figured they should)